### ISSUE TITLE
How to read and write platform clipboard?

### ISSUE THREAD

**Issue by AllenDang** [slint-ui/slint#2570]
I found that Platform has `get_clipboard` and `set_clipboard` function, but cannot figure out how to get a platform instance, any hint?

**Comment by ogoffart** [comment]
There is currently no way, with the public API, to get the Platform.  The idea is that the `Platform` and these functions are only used as a way for the Slint runtime to copy and paste (eg, from the TexInput)
But it is true that we may want to expose the Platform to users for when you want to do that in code.

What is your usecase and what are you trying to achieve?

**Comment by ogoffart** [comment]
I can see in the other issue (#2571) that you are trying to copy an image. But the current clipboard function can only do text.

Maybe what you want is using something like https://docs.rs/arboard/

**Comment by AllenDang** [comment]
Yes, I'm currently using arboard, but still happy to see slint has those functions. :P
